### DISCUSSION TITLE
How can I use css from external npm-package?

### DISCUSSION THREAD

**Discussion by liroliro** [codesandbox/codesandbox-client#6113] [Q&A]
Hey! As stated I was wondering if I can use external CSS in my examples on CSB? 
For example, in a "regular" HTML-project, I can just do a relative link to `node_modules`, from which I can get the file I want,
i.e `<link rel='stylesheet' href='node_modules/@npm-package/src/style/buttons.css' />`

Any thoughts or ideas how to work or circumvent this?

**Comment by JamesACS** [comment]
Hey @liroliro ! 

You can use external CSS, but the path you've put there as you've highlighted is a relative path. 

If the package you need has styles exported, that should be imported like any other style module (eg: import "@npm-package/src/style/buttons.css). 

If by regular, you mean you have a static HTML project you can just copy the CSS file next to the HTML one and use a relative link, or upload the CSS somewhere. 

If you want to rely on a published npm package, you can use a tool such as 'unpkg' to link directly to the file.
